docker/docker.env.sample

```diff
@@ -44,20 +44,20 @@ EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
 # SMTP email backend and settings
 #EMAIL_BACKEND='django.core.mail.backends.smtp.EmailBackend'
```

Comment:
nit: Looks like this variable is repeated - notice the space in line 43

Reply:
Thanks, I've removed the spaces around the equals sign. The variable is repeated to offer the user a choice of whether to use the smtp email backend or the console backend. I've added some wording in the file to clarify that.